src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/service/MakeAnApplicationTypesProvider.java

```diff
@@ -89,6 +105,21 @@ public DynamicList getMakeAnApplicationTypes(Callback<AsylumCase> callback) {
             case REASONS_FOR_APPEAL_SUBMITTED:
             case RESPONDENT_REVIEW:
             case SUBMIT_HEARING_REQUIREMENTS:
+
+                if (currentState == AWAITING_RESPONDENT_EVIDENCE
```

Comment:
I know this looks a bit clunky, but I couldn't put this block alone in the AWAITING_RESPONDENT_EVIDENCE option because I need the rest of the code (which is communal with the other states from line 100 to 107) to still run. 

I tried to put this block (lines 109-115) alone in the AWAITING_RESPONDENT_EVICENCE case and omitted the `break` keyword so that it'd also run the rest (lines 123-154), but code style checks don't allow fallthrough and flag it up as a code smell.